notebooks/263-latent-consistency-models-image-generation/263-latent-consistency-models-image-generation.ipynb

```diff
@@ -0,0 +1,1038 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    %pip install -q "torch" --index-url https://download.pytorch.org/whl/cpu
</pre><p>does it work on mac?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1390/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
yes, wheels for mac is also stored in the same folder